src/main/pages/portable-workspaces/creating-factories.adoc

```diff
@@ -7,66 +7,72 @@ permalink: creating-factories.html
+Sample factories:
 
-The Factory URL can include a branch or a subfolder. Here is an example of optional parameters:
+http://che.openshift.io/f?url=https://github.com/eclipse/che +
+http://che.openshift.io/f?url=https://github.com/eclipse/che/tree/language-server +
+http://che.openshift.io/f?url=https://gitlab.com/benoitf/simple-project
 
-* `?url=https://github.com/eclipse/che` che will be imported with master branch
-* `?url=https://github.com/eclipse/che/tree/5.0.0` che is imported by using 5.0.0 branch
-* `?url=https://github.com/eclipse/che/tree/5.0.0/dashboard` subfolder dashboard is imported by using 5.0.0 branch
+The factory URL can include a branch or a subdirectory. Following are examples of optional parameters:
+
+`?url=https://github.com/eclipse/che`::
+	Che is imported with the `master` branch.
+
+`?url=https://github.com/eclipse/che/tree/5.0.0`:: 
+	Che is imported by using the `5.0.0` branch.
+
+`?url=https://github.com/eclipse/che/tree/5.0.0/dashboard`:: 
+	Subdirectory `dashboard` is imported by using the `5.0.0` branch.
 
 [id="customizing-url-factories"]
-=== Customizing URL Factories
+=== Customizing URL factories
 
-There are 2 ways of customizing the runtime and configuration.
+Following are two ways to customize the runtime and configuration:
 
-*Customizing only the runtime*
+* Customizing only the runtime
 
-*Dockerfile works on Docker infra only!*
+* Dockerfile works on Docker infrastructure only
 
-Providing a `.factory.dockerfile` inside the repository will signal to the {{ site.product_mini_name }} URL Factory to use this Dockerfile for the workspace agent runtime. By default imported projects are set to a `blank` project type, however project type can be set in the `.factory.json` or workspace definition that the Factory inherits from.
+Providing a `.factory.dockerfile` inside the repository signals to the {{ site.product_mini_name }} URL factory to use this Dockerfile for the workspace agent runtime. By default, imported projects are set to a `blank` project type. You can also set the project type in the `.factory.json` or workspace definition that the factory inherits from.
 
-*Customizing the project and runtime* +
-Providing a `.factory.json` file inside the repository will signal to Che URL Factory to configure the project and runtime according to this configuration file. When a `.factory.json` file is stored inside the repository, any `Dockerfile` content is ignored as the workspace runtime configuration is defined inside the JSON file.
+*Customizing the project and runtime*
+
+Providing a `.factory.json` file inside the repository will signal to Che URL factory to configure the project and runtime according to this configuration file. When a `.factory.json` file is stored inside the repository, any `Dockerfile` content is ignored because the workspace runtime configuration is defined inside the JSON file.
```

Comment:
```suggestion
Providing a `.factory.json` file inside the repository signals to the Che URL factory to configure the project and runtime according to this configuration file. When a `.factory.json` file is stored inside the repository, any `Dockerfile` content is ignored because the workspace runtime configuration is defined inside the JSON file.
```